share/completions/s3cmd.fish

```diff
@@ -0,0 +1,121 @@
+# s3cmd
+
+function __s3cmd_is_remote_path
+	commandline -ct | string match -r -- "^s3://.*"
+end
+
+function __s3cmd_is_valid_remote_path
+	commandline -ct | string match -r -- "^s3://.*?/.*"
```

Comment:
Actually, the `?` in this context is the non-greedy modifier, it makes `.*` only match the bare minimum it needs for the regex to pass, i.e. `.*?/` against `foo/bar/` would match `foo` while `.*/` against `foo/bar/` would match `foo/bar`.